README.txt

```diff
@@ -1 +1,7 @@
 https://fedorahosted.org/fedoracommunity/wiki/Development
+
+Note: Some more dependencies to be installed:-
+
+$ sudo dnf -q -y install python-webhelpers fedmsg
```

Comment:
Out of principle I'm always a little against the ``-y`` option to give a chance to our users to review what they are doing.